### DISCUSSION TITLE
How does one calculate the Stride multiplication factor for calculating word timestamps for different models?

### DISCUSSION THREAD

**Discussion by ishansharma1320** [NVIDIA-NeMo/NeMo#5389] [Q&A]
As stated in this [comment](https://github.com/NVIDIA/NeMo/issues/4629#issuecomment-1200125199), the combination of stride multiplication factor along with model_stride is needed for the calculation of word-based timestamps.

Can someone please let me know how to calculate the aforementioned multiplication factor?

Lastly, is the stride multiplication factor dependent on the model architecture? 
if, yes, then do conformer_transducer_large and contextnet have the same value since their underlying architectures are the same?

**Comment by titu1994** [comment]
Window_stride is in the preprocessor config. Stride factor depends on model, there's no predefined section that defines it. 

It depends on the encoder, not the decoder - Conformer is usually 4x, ContextNet is 8x.
